docs/current_docs/api/arguments.mdx

```diff
@@ -458,67 +457,67 @@ The best way to understand this is with an example. Consider the following direc
 │   └── ...
 ```
 
-Here are two Dagger Functions with default contexts for their directory and file arguments:
+Here are two Dagger Functions with default paths for their directory and file arguments:
 
 <Tabs groupId="language">
 <TabItem value="Go">
 
-The default context is set by adding a `defaultPath` pragma on the corresponding Dagger Function argument.
+The default path is set by adding a `defaultPath` pragma on the corresponding Dagger Function `source` argument.
 
 ```go file=./snippets/functions/arguments-context-dir-file/go/main.go
 ```
 </TabItem>
 <TabItem value="Python">
 
-The default context is set by adding a `DefaultPath` annotation on the corresponding Dagger Function argument.
+The default path is set by adding a `DefaultPath` annotation on the corresponding Dagger Function `source` argument.
 
 ```python file=./snippets/functions/arguments-context-dir-file/python/main.py
 ```
 </TabItem>
 <TabItem value="TypeScript">
 
-The default context is set by adding an `@argument` decorator with a `defaultPath` parameter on the corresponding Dagger Function argument.
+The default path is set by adding an `@argument` decorator with a `defaultPath` parameter on the corresponding Dagger Function `source` argument.
 
 ```typescript file=./snippets/functions/arguments-context-dir-file/typescript/index.ts
 ```
 </TabItem>
 </Tabs>
 
-The following table describes how the arguments are resolved for differing values of the default context and differing directory types.
+The following table describes how the arguments are resolved for differing values of the default path and differing directory types.
 
 For a Git repository:
 
-| Default context ('defaultPath`) | Dagger Function invocation | `source` resolves to |
-| --- | --- | --- |
-| `/` | `dagger call -m my-module read-dir` | `/` |
-| `/src` | `dagger call -m my-module read-dir` | `/src` |
-| `.` | `dagger call -m my-module read-dir` | `/my-module` |
-| `..` | `dagger call -m my-module read-dir` | `/` |
-| `/README.md` | `dagger call -m my-module read-file` | `/README.md` |
-| `./README.md` | `dagger call -m my-module read-file` | Error: file does not exist |
-| `../README.md` | `dagger call -m my-module read-file` | `/README.md` |
+| Default path ('defaultPath`) | Context directory | Dagger Function invocation | `source` argument resolves to |
+| --- | --- | --- | --- |
+| `/` | `/` | `dagger call -m my-module read-dir` | `/` |
+| `/src` | `/` | `dagger call -m my-module read-dir` | `/src` |
+| `.` | `/` | `dagger call -m my-module read-dir` | `/` |
```

Comment:
Aren't `defaultPath` relative paths, relative to the root directory?

Reply:
In this case, since we assume it's a Git repository, my understanding is that the `defaultPath=.` resolves to the repository root

Reply:
@TomChv any comments or further changes needed here before merging?

Reply:
That can't be right. Is it @TomChv?

Reply:
> In this case, since we assume it's a Git repository, my understanding is that the defaultPath=. resolves to the repository root

No that's not the case? If you are inside a Git repository:
- `/` will resolve to the repo root
- `.` will resolve to the module's root dir (`dagger.json` location)